src/python/python-proto/python_proto/tests/strategies.py

```diff
@@ -33,16 +36,49 @@
     Label,
     MetricWrapper,
 )
-from python_proto.pipeline import Envelope, Metadata, Uuid
+from python_proto.pipeline import Envelope, Metadata, RawLog
+
+#
+# constants
+#
+# These values are used to parametrize the strategies defined below. Some of
```

Comment:
typo parameterize.

Reply:
Not a typo ;). The following spellings are all equivalent (and accepted):
- parameterize
- parametrize
- parameterise
- parametrise
The latter two are more often used in British English, the former two are used interchangeably (albeit, hopefully, consistently) in American English.